xcube_clms/constants.py

```diff
@@ -18,5 +18,114 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+import logging
+
+CLMS_API_URL = "https://land.copernicus.eu/api"
+
+# Configuration constants
+TIME_TO_EXPIRE = 24  # (in hours)
+RETRY_TIMEOUT = 60  # (in seconds)
 
 DATA_STORE_ID = "clms"
+
+PRELOAD_CACHE_FOLDER = "preload_cache/"
+
+# CLMS API Constants
+SEARCH_ENDPOINT = "@search"
+DOWNLOAD_ENDPOINT = "@datarequest_post"
+TASK_STATUS_ENDPOINT = "@datarequest_search"
+CANCEL_ENDPOINT = "@datarequest_delete"
+
+BATCH = "batching"
+NEXT = "next"
+
+PORTAL_TYPE = {"portal_type": "DataSet"}
+ACCEPT_HEADER = {"Accept": "application/json"}
+CONTENT_TYPE_HEADER = {"Content-Type": "application/json"}
+
+METADATA_FIELDS = "metadata_fields"
+FULL_SCHEMA = "fullobjects"
+
+DATA_ID_SEPARATOR = "|"
+
+# Dict keys
+
+DATA_ID_KEY = "data_id"
```

Comment:
Same for the following. Declare isolated where used.

Reply:
cc44bdb17